nixos/modules/services/torrent/qbittorrent.nix

```
@@ -0,0 +1,200 @@
+{
+  config,
+  pkgs,
+  lib,
+  utils,
+  ...
+}:
+let
+  cfg = config.services.qbittorrent;
+  inherit (builtins) concatStringsSep isAttrs isString;
+  inherit (lib)
+    literalExpression
+    getExe
+    mkEnableOption
+    mkOption
+    mkPackageOption
+    mkIf
+    maintainers
+    escape
+    collect
+    mapAttrsRecursive
+    ;
+  inherit (lib.types)
+    str
+    port
+    path
+    nullOr
+    unspecified
+    ;
+  inherit (lib.generators) toINI mkKeyValueDefault mkValueStringDefault;
+  gendeepINI = toINI {
+    mkKeyValue =
+      let
+        sep = "=";
+      in
+        k: v:
+        if isAttrs v then
+          concatStringsSep "\n"
+            (
+              collect
+                isString
+                (mapAttrsRecursive (path: value: "${escape [ sep ] (concatStringsSep "\\" ([ k ] ++ path))}${sep}${mkValueStringDefault {} value}") v)
+            )
+        else
+          mkKeyValueDefault { } sep k v;
+  };
+in
+{
+  options.services.qbittorrent = {
+    enable = mkEnableOption "qbittorrent, BitTorrent client.";
```

```suggestion
    enable = mkEnableOption "qbittorrent, BitTorrent client";
```

`mkEnableOption` adds the period already:

https://github.com/NixOS/nixpkgs/blob/a2a0316319cc0625ab75baba5084aacb272ba6d0/lib/options.nix#L98-L103

---

lol thats kinda funny